lua/autorun/client/cl_apple.lua

```diff
@@ -14,8 +14,8 @@ end
 
 net.Receive( "cfc_playerconnect_ajl", function()
     local name = net.ReadString()
-
-    chat.AddText( prefixColor, "• ", textColor, name .. " has connected to the server." )
+    local color = net.ReadColor()
+    chat.AddText( color, "• ", textColor, name .. " has connected to the server." )
```

Comment:
Wouldn't we just want their name to be rank-colored?
```suggestion
    chat.AddText( prefixColor, "• ", color, name, textColor, " has connected to the server." )
```

Reply:
good catch